### PR TITLE
ch(SASS  processing): Setup SASS processing on the client

### PR DESCRIPTION
**Pull request by EmmanuelBeja**
#### What does this PR do?
- Configure webpack to load SASS.
 - Write SASS code from mock ups.

#### Description of Task to be completed?
Setup SASS processing on the client

#### How should this be manually tested?
`
git clone https://github.com/andela/ah-jumanji-frontend.git && cd ah-jumanji-frontend && git checkout ch-SASS-processing-161966970
`
`
npm install
`
`
npm start
`

#### What are the relevant pivotal tracker stories?
[#161966970](https://www.pivotaltracker.com/story/show/161966970)

#### Screenshots (if appropriate)
Here is a screenshot of a sample page with bootstrap and our custom css precompiled from SASS.
screen shot 2019-01-09 at 21 29 04

#### Checklist:

- [x] My code follows the style guidelines of this project
- [ ] At least 2 people have reviewed my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,18 @@
 import {hot} from "react-hot-loader/root";
 import React from "react";
```
Comment (ghost) [inline src/App.js:1]: Consider using single quotes when importing for consistency

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
It is okay if the build fails on `CircleCI`. This is because the `.circleci/config.yml` does not exist in `develop` yet, as my branch, which introduces this file to `develop`, has not been merged yet.

File: src/App.js (hunk 1 above)
Comment (EmmanuelBeja) [reply src/App.js:1]: ow. that is actually code already on develop but i will implement that.

Comment (EmmanuelBeja) [reply src/App.js:1]: done.

**Review by GransonO — APPROVED** [review]
Smooth implementation. Just a rebase and we are good!

**Pull request merged**